src/components/tl/ucp/allreduce/allreduce.c

```diff
@@ -29,6 +29,10 @@ ucc_base_coll_alg_info_t
             {.id   = UCC_TL_UCP_ALLREDUCE_ALG_SLIDING_WINDOW,
              .name = "sliding_window",
              .desc = "sliding window allreduce (optimized for running on DPU)"},
+        [UCC_TL_UCP_ALLREDUCE_ALG_RING] =
+            {.id   = UCC_TL_UCP_ALLREDUCE_ALG_RING,
+             .name = "ring",
+             .desc = "ring-based allreduce (optimized for BW and simple topologies)"},
```

Comment:
At least for me, "simple topology" doesn't refer to something clear.
I would personally only keep "optimized for BW", or would use "ring topology"